File: crud-rubio/src/main/java/com/examen/crud_rubio/model/Person.java
package com.examen.crud_rubio.model;

import jakarta.persistence.*;

import java.math.BigDecimal;
import java.time.LocalDate;

@Entity
@Table(name = "person")
public class Person {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Integer id;

    private String nombre;

    private String apellido;

    private LocalDate fechaNacimiento;

    private String puesto;

    private BigDecimal sueldo;

    public Person() {
    }

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public String getApellido() {
        return apellido;
    }

    public void setApellido(String apellido) {
        this.apellido = apellido;
    }

    public LocalDate getFechaNacimiento() {
        return fechaNacimiento;
    }

    public void setFechaNacimiento(LocalDate fechaNacimiento) {
        this.fechaNacimiento = fechaNacimiento;
    }

    public String getPuesto() {
        return puesto;
    }

    public void setPuesto(String puesto) {
        this.puesto = puesto;
    }

    public BigDecimal getSueldo() {
        return sueldo;
    }

    public void setSueldo(BigDecimal sueldo) {
        this.sueldo = sueldo;
    }
}

File: Person/app/src/main/java/com/example/person/domain/model/Person.kt
package com.example.person.domain.model

import com.example.person.data.model.PersonDto

data class Person(
    val id: Int? = null,
    val name: String? = null,
    val lastName: String? = null,
    val birthdate: String? = null,
    val job: String? = null,
    val salary: Double? = null
)

fun Person.toPersonDto() = PersonDto(
    id = id,
    nombre = name,
    apellido = lastName,
    fechaNacimiento = birthdate,
    puesto = job,
    sueldo = salary
)